app/views/resources/index.html.erb

```diff
@@ -11,6 +11,12 @@
       name: "Documentation",
       description: "The official Hotwire Native docs." %>
 
+    <%= render "navigations/item",
+      path: book_url,
+      icon: "book-bold",
```

Comment:
This uses the same icon as the documentation link above. Do we want something different?

![RocketSim_Screenshot_iPhone_16_Pro_6 3_2025-04-23_10 58 36](https://github.com/user-attachments/assets/6cb29d1d-73e6-421c-b510-ed613295a26d)

Reply:
I think that works fine!